src/app/start/StartPage.tsx

```diff
@@ -0,0 +1,60 @@
+import { Link } from "react-router-dom";
+import StartButton from "../../components/Buttons/StartButton";
+import Card from "../../components/Card/Card";
+import styles from "./StartPage.module.scss";
+
+export default function StartPage() {
+  return (
+    <div className={styles.pageWrapper}>
+      <h1 className={styles.heading}>Witaj na Lesson4Future!</h1>
+      <section id="start-menu" className={styles.containerRows}>
+        <StartButton
+          iconClass="fa-solid fa-gamepad"
```

Comment:
nit: please use camelCase for classes, i know that css has standard of kebab-case but due to using mostly css modules we should keep consistency.